src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1547452866.inc.php

```diff
@@ -0,0 +1,118 @@
+<h1>Build #1547452866</h1>
+<h2>Date: 2019-01-14</h2>
+<div class="changelog">
+    - https://github.com/chameleon-system/chameleon-system/issues/265
+    - backend - main navigation: add new field icon_font_awesome
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+    ->setFields([
+        'cms_tbl_conf_id' => TCMSLogChange::GetTableId('cms_tbl_conf'),
+        'name' => 'new_field',
```

Comment:
Please cleanup this update by merging insert and update statement (there should be a single insert statement that creates the record, plus translations).